src/main/scala/com/evolutiongaming/scache/Cache.scala

```diff
@@ -81,14 +81,19 @@ object Cache {
   }
 
 
-  def loading[F[_] : Concurrent : Runtime, K, V]: Resource[F, Cache[F, K, V]] = {
+  def loading[F[_] : Concurrent : Runtime, K, V](partitions: Int): Resource[F, Cache[F, K, V]] = {
+    loading(Some(partitions))
+  }
+
+
+  def loading[F[_] : Concurrent : Runtime, K, V](partitions: Option[Int] = None): Resource[F, Cache[F, K, V]] = {
```

Comment:
could you please preserve backward compatibility ? leave old method and add a new one